keyboards/capsunlocked/cu80/v2_ansi/keymaps/via/keymap.c

```diff
@@ -0,0 +1,55 @@
+  [0] = LAYOUT_tkl_ansi(
+    KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,           KC_PSCR, KC_SLCK, KC_PAUS,
+
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_INS,  KC_HOME, KC_PGUP,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,  KC_END,  KC_PGDN,
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,                   KC_UP,
+    KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                             KC_RALT, KC_APP,  KC_RCTL, MO(1),            KC_LEFT, KC_DOWN, KC_RGHT
+  ),
+  [1] = LAYOUT_tkl_ansi(
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          RGB_M_P, RGB_M_B, RGB_M_R,
+
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_HUI, RGB_SAI, RGB_VAI,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_HUD, RGB_SAD, RGB_VAD,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, NK_TOGG, _______, _______, _______, _______,          _______,                   _______,
+    _______, _______, _______,                   _______,                            _______, _______, _______, _______,          RGB_RMOD,RGB_TOG, RGB_MOD
+  ),
+  [2] = LAYOUT_tkl_ansi(
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
+
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                   _______,
+    _______, _______, _______,                   _______,                            _______, _______, _______, _______,          _______, _______, _______
+  ),
+  [3] = LAYOUT_tkl_ansi(
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
```

Comment:
```suggestion
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
```

Reply:
I gather this is because the bootmagic feature is enabled therefore there is no need for a binding on the layers?

Reply:
`RESET` already lives on layer 1, that was just a cut and paste error on my part.